src/scripting/KM_ScriptingStates.pas

```diff
@@ -1044,17 +1045,21 @@ function TKMScriptStates.StatResourceProducedMultipleTypesCount(aPlayer: Byte; a
 end;
 
 
-//* Version: 4758
+//* Version: 4758 Update 7000+
 //* Get players color as text in hex format
 //* Result: Player color
-function TKMScriptStates.PlayerColorText(aPlayer: Byte): AnsiString;
-begin
-  try
+function TKMScriptStates.PlayerColorText(aPlayer: Byte;
+  bLight: Boolean): AnsiString;

```

Comment:
We use 'a' prefix for parameters, so pls change 'bLight' to 'aLight'.
Also why did you start new line here ?